src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -0,0 +1,564 @@
+require "yast"
+require "cwm"
+require "y2storage"
+
+module Y2Partitioner
+  module Widgets
+    # Fstab mixin
+    module FstabCommon
+      def initialize(options)
+        textdomain "storage"
+
+        @options = options
+      end
+
+      def supported_by_filesystem?
+        return false if !@options.filesystem_type
+
+        if respond_to?("supported_filesystems")
+          supported_filesystems.include?(@options.filesystem_type.to_sym)
+        else
+          self.class.const_get("VALUES").all? do |v|
+            @options.filesystem_type.supported_fstab_options.include?(v)
+          end
+        end
+      end
+
+      def draw_widget?(widget)
+        return true if widget.supported_by_filesystem?
+
+        false
+      end
+
+      def draw(widget)
+        return Empty() unless draw_widget?(widget)
+
+        Left(widget)
+      end
+
+      def draw_with_vspace(widget)
+        return [Empty()] unless draw_widget?(widget)
+
+        [Left(widget), VSpacing(1)]
+      end
+
+      def delete_from_fstab!(option)
+        @options.fstab_options.delete_if { |o| o =~ option }
```

Comment:
it have to be documentated, that option is regexp and not string and also notice possible issue about substring.